vue/components/ui/molecules/calendar/calendar.vue

```diff
@@ -0,0 +1,296 @@
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 10px 10px 10px;
+}
+
+.calendar > .calendar-header > .header-center {
+    align-items: center;
+    display: inline-block;
+    display: flex;
+}
+
+.calendar > .calendar-content {
+    display: flex;
+    font-weight: 400;
+    height: 100%;
+    padding: 0px 20px 10px 20px;
+}
+
+.calendar > .calendar-content .calendar-table {
+    width: 100%;
+}
+
+.calendar > .calendar-content .calendar-table .table-header .row {
+    height: 40px;
+}
+
+.calendar > .calendar-content .calendar-table .table-header .row .heading {
+    font-weight: 400;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell {
+    height: 40px;
+    padding: 0px;
+    text-align: center;
+    width: 40px;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell.clickable {
+    cursor: pointer;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell .circle {
+    border: 3px solid transparent;
+    border-radius: 50%;
+    box-sizing: border-box;
+    color: #1d2631;
+    height: 40px;
+    line-height: 35px;
+    margin: auto;
+    user-select: none;
+    width: 40px;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell.clickable:not(.selected):hover .circle {
+    background-color: #ecf0f3;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell:not(.clickable) .circle {
+    color: #bac2cb;
+}
+
+.calendar > .calendar-content .calendar-table .table-body .row .cell.selected .circle {
+    background-color: #1d2631;
+    border: 3px solid #ecf0f3;
+    color: #ffffff;
+}
+
+.calendar .select ::v-deep .select-container .select-button {
+    background-color: #ffffff;
+}
+</style>
+
+<script>
+export const Calendar = {
+    props: {
+        /**
+         * The value of input date in String or Date format.
+         * Example: "2020/12/31"
+         */
+        value: {
+            type: Date | String,
+            default: null
+        },
+        /**
+         * Month labels to be used directly starting from January.
+         */
+        monthLabels: {
+            type: Array,
+            default: () => [
+                "January",
+                "February",
+                "March",
+                "April",
+                "May",
+                "June",
+                "July",
+                "August",
+                "September",
+                "October",
+                "November",
+                "December"
+            ]
+        },
+        /**
+         * Week labels to be used directly starting from Sunday.
+         */
+        weekLabels: {
+            type: Array,
+            default: () => ["Su", "Mo", "Tu", "We", "Th", "Fr", "Sa"]
+        }
+    },
+    data: function() {
+        return {
+            valueData: this.value,
+            calendarVisibility: true,
+            day: 1,
+            month: new Date().getMonth(),
+            year: new Date().getFullYear()

```

Comment:
Shouldn't this be based on `value`?

Reply:
this is in case no initial date (`value`) was received from props but the calendar will start showing the current month and current year

<img width="377" alt="imagem" src="https://user-images.githubusercontent.com/24736423/123775190-11e0c700-d8c6-11eb-885d-1632d760433c.png">